general-concepts/mirrors/text.xml

```diff
@@ -128,14 +128,19 @@ where <c>myname</c> refers to the username of the developer.
 
 <body>
 <p>
-To manually upload a file to <c>mirror://gentoo</c>, <c>scp</c> it to
-<c>dev.gentoo.org:/space/distfiles-local</c>. The file should appear
-on the mirrors within four hours (note that this is <e>less
-frequent</e> than <uri link="::general-concepts/git-to-rsync"/>).
+Historically Gentoo developers were able to sync distfiles directly to mirrors
+via <c>dev.gentoo.org:/space/distfiles-local</c> which enabled <c>
+mirror://gentoo</c> syntax to be used in an ebuild. This has been prohibited
+since due to issues listed above, and Gentoo developers must now utilize their
+own devspace to host files. Non-developer contributors are encouraged to use
+their own GitHub / GitLab space, or contact a Gentoo developer willing to host
+the distfile.
+</p>
 
+<p>
 If the upstream download location for a package uses a non-standard TCP port
-(anything other than 21, 80 or 443), you <e>must</e> manually mirror the files. Not
-doing so can cause all kinds of problems with strict firewalls.
+(anything other than 21, 80 or 443), you <e>must</e> manually mirror the files.
+Not doing so can cause all kinds of problems with strict firewalls.
 </p>
 </body>
```

Comment:
I wonder if we shouldn't drop the entire section. The second paragraph could be added at the end of the previous section.

I'd drop the paragraph about `/space/distfiles-local` altogether. It is only a historical curiosity which we need not document.

Reply:
I really thought about it as well, but though it was nice to explain where possible `mirrors://gentoo` came from. Yeah, I can definitely combine the sections.